miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/AbstractST.java

```diff
@@ -139,6 +152,93 @@ protected String pollingResponse(String url) throws Exception {
     return response;
   }
 
+  protected <T> void abstractTestBulkCreateAsync(String controllerBase, Class<T> createType, List<?> dtos)
+      throws Exception {
+
+
+    MvcResult mvcResult = getMockMvc()
+        .perform(post(controllerBase + "/bulk").contentType(MediaType.APPLICATION_JSON).content(makeJson(dtos)))
+        .andExpect(status().isAccepted())
+        .andReturn();
+
+    String id = JsonPath.read(mvcResult.getResponse().getContentAsString(), "$.operationId");
+    String response = pollingResponse(controllerBase + "/bulk/" + id);
+
+    Integer id1 = JsonPath.read(response, "$.data[0].id");
+    Integer id2 = JsonPath.read(response, "$.data[1].id");
+
+    assertNotNull(currentSession().get(createType, id1));
+    assertNotNull(currentSession().get(createType, id2));
```

Comment:
This implementation requires you to supply two (and only two) dtos to the function. Can you make this support any number of DTOs? (Same with abstractTestBulkUpdateAsync below)